cluster/core/src/main/scala/com/lightbend/lagom/internal/cluster/JoinClusterImpl.scala

```diff
@@ -10,14 +10,19 @@ import akka.actor.CoordinatedShutdown._
 import akka.cluster.Cluster
 import akka.management.cluster.bootstrap.ClusterBootstrap
 import com.lightbend.lagom.internal.akka.management.AkkaManagementTrigger
+import com.typesafe.config.Config
 import play.api.{ Environment, Mode }
 
 import scala.concurrent.Future
 
 private[lagom] object JoinClusterImpl {
 
+  @deprecated("Use join(ActorSystem, Environment, Config, AkkaManagementTrigger) instead", "2.0.0")
```

Comment:
Assuming for now that the next version is 2.0.0.

Reply:
The current version in `master` is 1.6.0-SNAPSHOT. Shall we move it to 2.0.0-SNAPSHOT?

Personally, I would prefer to keep it as 1.6.0 as long as we don't have it clearly deifned.

Reply:
Now that's clear that next version will be 1.6.0, we should change that message.